docs/ops/generation/RandomUniform_8.md

```diff
@@ -0,0 +1,195 @@
+## RandomUniform <a name="RandomUniform"></a> {#openvino_docs_ops_generation_RandomUniform_8}
+
+**Versioned name**: *RandomUniform-8*
+
+**Category**: Generation
+
+**Short description**: *RandomUniform* operation generates a sequence of random values from a uniform distribution.
+
+**Attributes**:
+
+* *output_type*
+
+    * **Description**: the type of the output.
+    * **Range of values**: "i32", "i64", "f16", "f32", "f64".
+    * **Type**: string
+    * **Default value**: "f32"
+    * **Required**: *Yes*
```

Comment:
How that's possible that attribute is required and at the same time has default value?
I think either default value should be `None` or required should be `*No*`

Reply:
Fixed.